README.md

```diff
@@ -1,2 +1,61 @@
-# openbmp-python-api-message
-Python Library for parsing OpenBMP Kafka messages
+# OpenBMP API Message Python Library
+
+This library implements the OpenBMP message bus specification as defined at [MESSAGE_BUS_API.md](http://openbmp.org/#!docs/MESSAGE_BUS_API.md).
+ 
+Messages from kafka (headers and content) are parsed and made available in ```Map``` and String/JSON.
+> ### API supports Python 2.7
+> ### Current Schema supported is 1.3
+
+## Usage
+ 
+Use this library in any existing python project.
+
+### Code
+
+```python
+from api import *
+
+raw_kafka_message_data = '...';
+
+msg = Message(raw_kafka_message_data)
+
+#Parse the content data by its type
+peer = Peer(msg.getContent());
```

Comment:
Update to include version or just the message class object instead of msg.getContent().